### PR TITLE
Start the API wrapper with the first function

### PR DESCRIPTION
**Pull request by ms10398**
PIng @kozlown @teolemon

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "url": "git+https://github.com/ms10398/openfoodfacts-nodejs.git"
+  },
+  "keywords": [
+    "OFF",
+    "OpenFoodFacts",
+    "NodeJS",
+    "Wrapper"
+  ],
+  "author": "ms10398",
+  "license": "Apache-2.0",
```
Comment (kozlown) [inline package.json]: I think that we should use the same licence and author than [the OFF server](https://github.com/openfoodfacts/openfoodfacts-server)

Comment (ms10398) [reply package.json]: - `LICENSE` was added by @teolemon to the repo thats why I chose to kept it.

- and about author I will change it in few mins.

Thanks for the review.

Just give me feedback about the License than I would change the repository `LICENSE` too.

**Comment by ms10398** [comment]
@kozlown I have changed the author accordingly

:slightly_smiling_face:

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,381 @@
+{
```
[the remark this answers is not in the document]
Comment (ms10398) [reply package-lock.json]: I prefer npm more as I have worked on it a lot.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+## Examples
+
+For example check [here](example/examples.js)
+
+## Contribute
+
+Fork the repository and feel free to send a PR.
+
+## Author
```
[the remark this answers is not in the document]
Comment (ms10398) [reply README.md]: Yes I will surely love to implement it that way :slightly_frowning_face:

File: package-lock.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (ms10398) [reply package-lock.json]: Yes sure will add it in few mins

**Comment by ms10398** [comment]
@kozlown Made the suggested changes :slightly_smiling_face:

**Pull request merged**